content/src/service/synchronization/batchDeployer.ts

```diff
@@ -110,14 +112,19 @@ export function createBatchDeployerComponent(
             successfulDeployments.add(entity.entityId)
             deploymentsMap.delete(entity.entityId)
           } catch (err: any) {
-            // failed deployments are automatically rescheduled
-            components.deployer.reportErrorDuringSync(
-              entity.entityType as any,
-              entity.entityId,
-              FailureReason.DEPLOYMENT_ERROR,
-              entity.authChain,
-              err.toString()
+            const errorDescription = err.toString()
+            logs.warn(
+              `Deployment of entity (${entity.entityType}, ${entity.entityId}) failed. Reason was: '${errorDescription}'`
             )
```

Comment:
Prefer structured logs
```suggestion
            const errorDescription = err.toString()
            logs.warn(`Entity deployment failed`, {
              entityType: entity.entityType,
              entityId: entity.entityId,
              reason: errorDescription
            })
```